get_alerts.py

```diff
@@ -25,14 +70,63 @@ def include_alert(alert, args):
     Determine whether alert should be included
     Return True if included, False otherwise
     '''
-    title = alert.get('title')
-    if not title:
-        return True
-    if args.startswith:
-        return title.startswith(args.startswith)
-    if args.contains:
-        return args.contains in title
-    return True
+    #This is a legacy section of code
```

Comment:
We can just delete this commented out part. It will be in source control if we ever want to get it back :)